Cargo.toml

```diff
@@ -37,3 +39,4 @@ solana-rpc-client = { git = "https://github.com/Lightprotocol/agave", branch = "
 solana-rpc-client-api = { git = "https://github.com/Lightprotocol/agave", branch = "v1.18.11-enforce-cpi-tracking" }
 solana-runtime = { git = "https://github.com/Lightprotocol/agave", branch = "v1.18.11-enforce-cpi-tracking" }
 solana-sdk = { git = "https://github.com/Lightprotocol/agave", branch = "v1.18.11-enforce-cpi-tracking" }
+solana-transaction-status = { git = "https://github.com/Lightprotocol/agave", branch = "v1.18.11-enforce-cpi-tracking" }
```

Comment:
what is switching back to the main agave repo contingent on?

Reply:
On this commit being released:

https://github.com/anza-xyz/agave/pull/1504

Without it, our program tests can't track inner instructions, so we wouldn't be able to test the event emission.